microprofile/WFLY-14148-microprofile-opentracing-move.adoc

```diff
@@ -0,0 +1,58 @@
+= WFLY-14148 Remove MicroProfile Opentracing from WildFly EE
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.redhat.com/browse
+
+== Overview
+
+This is about removing the MicroProfile Opentracing functionalities from WildFly EE
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/WFLY-14156[WFLY-14148]
+
+=== Related Issues:
+
+
+=== Dev Contacts:
+
+* mailto:ehugonne@redhat.com[Emmanuel Hugonnet]
+
+=== QE Contacts:
+
+* TBD
+
+=== Affected Projects or Components:
+
+* WildFly
+
+== Requirements
+
+
+* The MicroProfile Opentracing functionality is provided in the full 'wildfly' feature pack only
+* Adding a new 'microprofile-opentracing' layer in the microprofile galleon pack and adding it to the microprofile 'observability' layer.
+* Removing it from wildfly ee feature pack 'observability' layer.
+* Adding a feature_group to cover the default configuration with a Jaeger tracer.
+
```

Comment:
Please add that testsuite modules outside of the 'testsuite/integration/microprofile' and 'testsuite/integration/microprofile-tck' will not require MicroProfile OpenTracing to be present to run. Any tests that do will be moved to testsuite/integration/microprofile.